common/djangoapps/edxmako/shortcuts.py

```diff
@@ -87,13 +87,14 @@ def marketing_link(name):
         # don't try to reverse disabled marketing links
         if link_map[name] is not None:
             host_name = get_current_request_hostname()
-            if all([host_name and 'edge' in host_name, 'http' in link_map[name]]):
+            if link_map[name].startswith('http'):
                 return link_map[name]
             else:
                 try:
                     return reverse(link_map[name])
                 except NoReverseMatch:
-                    raise Http404
+                    log.debug(u"Cannot find corresponding link for name: %s", name)
+                    return '#'
```

Comment:
So we just end up with `<a href="#">Blah</a>` in a rendered page somewhere?

I wonder if there's some way we can surface this as a warning that we'll see, but won't pollute the logs. Some kind of metric?